docs/installation.md

```diff
@@ -134,34 +149,41 @@ Create a Service with type `LoadBalancer` using the appropriate manifest for you
 ## Upgrading NGINX Gateway Fabric
 
 > **Note**
+>
 > See [below](#configure-delayed-termination-for-zero-downtime-upgrades) for instructions on how to configure delayed
 > termination if required for zero downtime upgrades in your environment.
 
 ### Upgrade NGINX Gateway Fabric from Manifests
 
 1. Upgrade the Gateway Resources
 
-   Before you upgrade, ensure the Gateway API resources are the correct version as supported by the NGINX Gateway
-   Fabric - [see the Technical Specifications](/README.md#technical-specifications).
-   The [release notes](https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.8.1) of the new version of the
-   Gateway API might include important upgrade-specific notes and instructions. We advise to check the release notes of
-   all versions between the one you're using and the new one.
+    Before you upgrade, ensure the Gateway API resources are the correct version as supported by the NGINX Gateway
+    Fabric - [see the Technical Specifications](/README.md#technical-specifications).
+    The [release notes](https://github.com/kubernetes-sigs/gateway-api/releases/tag/v1.0.0) of the new version of the
+    Gateway API might include important upgrade-specific notes and instructions. We advise to check the release notes of
+    all versions between the one you're using and the new one.
 
-    To upgrade the Gateway resources from [the Gateway API repo](https://github.com/kubernetes-sigs/gateway-api), run:
+    To upgrade the Gateway CRDs from [the Gateway API repo](https://github.com/kubernetes-sigs/gateway-api), run:
 
     ```shell
-    kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v0.8.1/standard-install.yaml
+    kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.0.0/standard-install.yaml
     ```
 
-1. Upgrade the NGINX Gateway Fabric CRDs
+    If you are running on Kubernetes 1.23 or 1.24 you also need to update the validating webhook. To do so, run:
```

Comment:
Should we include a note here about deleting the webhook if you have it running in k8s 1.25+?

Reply:
added: https://github.com/nginxinc/nginx-gateway-fabric/pull/1250/commits/29e7271528e17ae65754f9a06afb4723997e2fd2